### ISSUE TITLE
Broken Ubuntu shortcut STRG+Alt+P not working.

### ISSUE THREAD

**Issue by nextgenthemes** [mehcode/atom-project-plus#66]
Broken Ubuntu shortcode STRG+Alt+P not working.

**Comment by willdawsonme** [comment]
@nextgenthemes Do you know how international keyboard bindings resolve inside of Atom? If you could, toggle the keybinding resolver in Atom and see what `strg+alt+p` resolves to. There is an update to atom-keymap which should be release in atom core as a beta in the next week or so (https://github.com/atom/atom/pull/12697), so fingers crossed that helps. Your German keyboard layout isn't easy to test for without access to a German keyboard, so any help or PRs would be appreciated!

**Comment by nextgenthemes** [comment]
There is nothing international about STRG+ALT+P you call it CTRL+ALT+P was not thinking.

I installed the 1.11.02-beta5 anyway, noting changed.

Isn't the keyboard resolver popping up automatically, how do I use or trigger it? I two shortcuts showing up if i type ctrl-alt-p into the atom shortcuts thing, the one from project plus and this. Currently none work. This 'body' seems wrong to me. Isn't this supposed to be the platform? 

```
'body':
  'ctrl-alt-p': 'window:run-package-specs'
```

Maybe I have another package installed that messes with it? Can you tell me what this is. Does atom have this shortcut bind by default?

OK I now have overwritten both shortcuts, both "work" now. Still nothing shows up with the project finder.

I just reinstalled this package to help, installed the beta ... I was using project switcher, just putting symlink to my projects in one dir and setting that to be used and it it works great. I will probably abandon this again. I have the feeling nobody ever tested this on a linux PC and if the shortcut is a atom default one I think the project should not use it.

**Comment by willdawsonme** [comment]
You're right, `ctrl-alt-p` does seem to be a core keybinding. You can toggle the keybinding resolver with `ctrl-.` on linux. Could you check that `alt-shift-p`, `alt-shift-tab` and `ctrl-alt-shift-tab` are currently unbound on linux? As far as I can tell, they're unbound and would be a good alternative.
